<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tttTimelapseManager</class>
 <widget class="QDockWidget" name="tttTimelapseManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>70</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>Timelapse</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="firstButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons/icons.qrc">
        <normaloff>:/Icons/begin.png</normaloff>:/Icons/begin.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backwardButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons/icons.qrc">
        <normaloff>:/Icons/previous.png</normaloff>:/Icons/previous.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="forwardButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons/icons.qrc">
        <normaloff>:/Icons/next.png</normaloff>:/Icons/next.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lastButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Icons/icons.qrc">
        <normaloff>:/Icons/end.png</normaloff>:/Icons/end.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="text">
    <string>Backward</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="text">
    <string>Last</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>firstButton</sender>
   <signal>clicked()</signal>
   <receiver>actionFirst</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastButton</sender>
   <signal>clicked()</signal>
   <receiver>actionLast</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>actionForward</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backwardButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBackward</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
